eigentrust-zk/src/circuits/dynamic_sets/mod.rs

```diff
@@ -514,80 +514,52 @@ impl<const NUM_NEIGHBOURS: usize, const NUM_ITER: usize, const INITIAL_SCORE: u1
 					&config.main,
 					layouter.namespace(|| "op_score_sum == 0"),
 				)?;
-				for j in 0..NUM_NEIGHBOURS {
-					let op_pk_j_x = op_pk_x_i[j].clone();
-					let op_pk_j_y = op_pk_y_i[j].clone();
 
-					// Condition 1. op_pk_j != pk_i
-					let equal_chip = IsEqualChipset::new(op_pk_j_x.clone(), pk_i_x.clone());
-					let is_pk_i_x = equal_chip.synthesize(
+				for j in 0..NUM_NEIGHBOURS {
+					let addr_j = set[j].clone();
+					// Condition 1. addr_j != addr_i
+					let equal_chip = IsEqualChipset::new(addr_j.clone(), addr_i.clone());
+					let is_add_i = equal_chip.synthesize(
```

Comment:
I think the name should be `is_addr_i`, not the `is_add_i`.
right? @4rmanc @lazovicff